lms/views/api/blackboard/sync.py

```diff
@@ -68,7 +92,7 @@ def _get_blackboard_groups(self):
         if not groups:
             raise CanvasGroupSetEmpty(group_set=group_set_id)
 
-        return groups
+        return self._to_groups_groupings(groups)
```

Comment:
We are are already returning `Grouping` now.

Reply:
Note that `_to_groups_groupings` uses `upsert_canvas_group`. That's wrong, needs to be adapted for BB.